src/helpers/strategies.ts

```diff
@@ -21,24 +20,31 @@ async function loadStrategies() {
     return true;
   }
 
-  Object.values(spaces).forEach((space: any) => {
-    const ids = new Set<string>(space.strategies.map(strategy => strategy.name));
-    ids.forEach(id => {
-      if (res[id]) {
-        res[id].spacesCount = (res[id].spacesCount || 0) + 1;
-      }
-    });
-  });
+  const results = new Map(
+    (
+      await db.queryAsync(`
+        SELECT
```

Comment:
This can be an heavy request isn't? How long does it take to resolve? How often is it called?

Reply:
~40ms to resolve, on my dataset of 13k spaces.

It's still not 100% live query, as it's only run along the fetch strategies query from score url, so once every 1min

Reply:
Ok

Reply:
Do you want more refactoring, and use 100% live counters (and not cached for 1min) ? 

This will bring more consistency, as no other counters are cached and lagging behind.

By enforcing only known strategies when submitting via sequencer, we could also get rid of the condition, checking if the strategy is a valid one from score-api, and have true live data from the strategies operation. (can also get rid of the /helpers/startegies.ts file)

Reply:
I prefer not using such query as live query, also not sure how you wouod get ride of strategies helpers, this is loading strategies metadata that are not available from the hub

Reply:
true, forgot about the other strategies data. We get less than 1 request/min on the hub for the strategies object, should have very minimal impact on performance

Reply:
Takes around 2.8s on prod, will increase when we get more space 🤔